src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -29,6 +29,18 @@ describe('<Tooltip />', () => {
     expect(screen.queryByTestId('tooltip-content')).not.toBeInTheDocument();
   });
 
+  it('should close tooltip via escape key for disabled buttons', async () => {
+    // disable animation for test
+    render(<InteractiveDisabled duration={0} />);
+    const trigger = await screen.findByTestId('disabled-child-tooltip-wrapper');
+    expect(screen.queryByTestId('tooltip-content')).not.toBeInTheDocument();
+    await userEvent.hover(trigger);
+    expect(screen.getByTestId('tooltip-content')).toBeInTheDocument();
+    // TODO: make worky
+    // await userEvent.keyboard('{esc}');
+    // expect(screen.queryByTestId('tooltip-content')).not.toBeInTheDocument();
```

Comment:
This is the test that isn't working 🤔 Could it be something about using a keyboard event with a span? That's the only thing I can think of..

Reply:
You have to pass `duration` as an arg to the story i.e.:

```
render: (args) => (
...
        <Tooltip
          duration={args.duration}
...
```

Reply:
Oh nice! Thanks! ❤️